app/actions/app-data.js

```diff
@@ -21,11 +21,14 @@ export function updateUser({user, id}) {
     };
 }
 
-export function fetchData() {
+export function fetchData(username) {
+    if (username === undefined) {
+        // TODO: implement error handling
+    }
     return function (dispatch) {
         dispatch(fetchingData());
 
-        return getUsers()
+        return getUsers(username)
```

Comment:
I would rather create a separate method, getUser. As this is should be for getting all the users.
Well we can argue though whether this method should exist...